helm/prometheus-rules/templates/alerting-rules/cert-manager.rules.yml

```diff
@@ -17,7 +17,7 @@ spec:
           If memory usage value is equal to memory limit value then it is likely the pod will be evicted.
           If no limits are set then the pod will burst.
           `}}
-      expr: sum by (pod, namespace, container) (container_memory_working_set_bytes{container=~"(cert-manager)"}) / sum by (pod, namespace, container) (kube_pod_container_resource_requests_memory_bytes{container=~"(cert-manager)"}) >= 0.85
+      expr: sum by (cluster_id, pod, namespace, container) (container_memory_working_set_bytes{container=~"(cert-manager|cert-manager-app-controller)"}) / sum by (cluster_id, pod, namespace, container) (kube_pod_container_resource_requests{resource="memory", unit="byte",container=~"(cert-manager|cert-manager-app-controller)"}) >= 0.85
```

Comment:
This fixes the cert-manager alert for newer versions

Reply:
But do we run this new version all around? How can we be assured of that?

Reply:
This is the version fix "cert-manager|cert-manager-app-controller"

Reply:
This kube_pod_container_resource_requests was added in ksm 2.0.0

Reply:
And according to this we do not run ksm older than 2.6 https://giantswarm.grafana.net/explore?schemaVersion=1&panes=%7B%229rq%22:%7B%22datasource%22:%22000000006%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22aggregation:container:images%7Bimage%3D~%5C%22.%2Akube-state-metrics.%2Av1.%2A%5C%22%7D%22,%22range%22:true,%22instant%22:true,%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%22000000006%22%7D,%22editorMode%22:%22code%22,%22legendFormat%22:%22__auto%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D%7D&orgId=1